views/flix.ejs

```diff
@@ -1,32 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
     <h1>FindAFlick</h1>
     <ul>
-    <% flix.forEach( el => { %>
+        <% flix.forEach( el=> { %>
             <li class='flixItem' data-id='<%=el._id%>'>
-                <span class='<%= el.completed === true ? 'completed' : 'not'%>'><%= el.flix %></span>
+                <span class='<%= el.completed === true ? ' completed' : 'not' %>'><%= el.flix %></span>
                 <span class='del'> Delete </span>
             </li>
-    <% }) %>    
+            <% }) %>
     </ul>
 
-    <h2><%= user.userName %> has <%= left %> movies & TV shows to watch.</h2>
-
-    <form action="/flix/createFlix" method='POST'>
-        <input type="text" placeholder="Enter Movie/Tv Show" name='flixItem'>
-        <input type="submit">
-    </form>
+    <input id='text-movie' type="text" placeholder="Enter Movie/Tv Show" name='flixItem'>
+    <button id='search-movie'>Submit</button>
 
     <a href="/logout">Logout</a>
+    <section class="movieInfo displayNone">
+        <div class="container">
+            <div class="posterContainer">
+                <img id ="moviePoster" src="" alt="" class="poster">
+            </div>
+            <div>
+                <h2 id="movieTitle"></h2>
+                <p>Release date: <a id="movieYear"></a></p>
+                <p>Director(s): <a id="movieDirectors"></a></p>
+                <p>Writer(s): <a id="movieWriters"></a></p>
+                <p>Lead Actors: <a id="movieLeadActors"></a></p>
+                <p>IMDB Rating: <a id="movieIMDB"></a></p>
+                <p>Rotten Tomatoes Rating: <a id="movieTomatoes"></a></p>
+            </div>
+            <div class="synopsisContainer">
+                <h3>Synopsis</h3>
+                <p id="movieSynopsis"></p>
+            </div>
+        </div>
+    </section>
+
```

Comment:
I was thinking this could be created as its own component rather than putting it on this page, but not a blocker for now, it can be another pr

Reply:
It can be extracted out at any point, and probably should. My intent for this PR was to include a minimal amount of HTML mark-up in order to display the data returned from the server. And at that point I figured to add some rudimentary layout so that it wouldn't burn my eyes (as much).